src/osvr/Client/DisplayConfig.cpp

```diff
@@ -160,5 +191,38 @@ namespace client {
         }
         return true;
     }
+
+    OSVR_DisplayInputCount DisplayConfig::getNumDisplayInputs() const {
+
+        // assume there's 1 input but still check if there are more than one
+        // display inputs by checking every viewer eye (0 based index)
```

Comment:
This is conceptually a little flawed - there should be an object called VideoInput or something like that, that has properties width and height, and that the display config owns a container of. Once we get more data on video inputs, we'll need that.

You're not assuming there is one input, so that bit of the comment can be removed.